packages/components/sronly/stories/SrOnly.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { SectionHeading } from '@contentful/f36-typography';
+import { SrOnly } from '../src/SrOnly';
+import { TextLink } from '@contentful/f36-text-link';
+import { Stack } from '@contentful/f36-core';
+
+export default {
+  title: 'Components/SrOnly',
```

Comment:
@Lelith one more thing that came to my mind, is that maybe we could have a separate section in storybook, something like "helpers" or similar? WDYT?